src/libraries/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -14,6 +14,7 @@ public enum Architecture
         Arm64 = 3,
         Wasm = 4,
         S390x = 5,
+        Loongarch64 = 6,
```

Comment:
These are adding a new public APIs, so it may be a good idea to run them through a quick API review. Could you please file API proposal issue for it? https://aka.ms/api-review has details.

I am not sure about the casing. Should it be `Loongarch64` or `LoongArch64` ?

Reply:
LoongArch64 is better, but it's not so strict.
I will update  to LoongArch64.